pkg/client/clientset/versioned/scheme/register.go

```diff
@@ -48,5 +48,8 @@ func init() {
 // After this, RawExtensions in Kubernetes types will serialize kube-aggregator types
 // correctly.
 func AddToScheme(scheme *runtime.Scheme) {
-	scalingv1alpha1.AddToScheme(scheme)
+	err := scalingv1alpha1.AddToScheme(scheme)
+	if err != nil {
```

Comment:
should we do something with the error? if its not going to happen `// nolint:errcheck`